src/btree/bt_prefetch.c

```diff
@@ -100,7 +100,7 @@ __wt_prefetch_page_in(WT_SESSION_IMPL *session, WT_PREFETCH_QUEUE_ENTRY *pe)
         WT_ERR(__wt_page_in(session, pe->ref, WT_READ_PREFETCH));
         WT_ERR(__wt_page_release(session, pe->ref, 0));
     } else
-        ret = WT_ERROR;
+        return (WT_ERROR);
```

Comment:
I thought we'd have a dist script that fails with this direct return when there is an error handling. i.e: once a WT_ERR is used all subsequent function exits need to go via the error handling path.

Reply:
Hmm, interesting. I didn't seem to get any errors with s_all either... I thinking we could even get rid of the `err:` label altogether since it doesn't actually look like it's doing any special error handling logic.